### ISSUE TITLE
Part Scale: Keyboard navigation is broken.

### ISSUE THREAD

**Issue by leoheck** [FreeCAD/FreeCAD#23772]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Problem description

All the widgets should have keyboard navigation reviewed.

This is one example of it.
The whole thing is impossible to navigate in a sane way.
A special problem is when scaling Non-Uniformly.
I cannot navigate to the next item easily.

From the X factor, the selection disappears when hitting tab,  and one more click leads me to the Z factor.

<img width="425" height="444" alt="Image" src="https://github.com/user-attachments/assets/6d58ea57-b504-4ca9-91bf-eba5d1fc9e9e" />

### Full version info

```shell
1.1-dev (probably since forever)
```

### Subproject(s) affected?

Core

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by FEA-eng** [comment]
> From the X factor, the selection disappears when hitting tab, and one more click leads me to the Z factor.

Small correction - it doesn't disappear, it highlights the "Non-uniform scaling" radio button. Otherwise, the issue is still relevant.

```
OS: Windows 10 build 19045
Architecture: x86_64
Version: 1.1.0dev.20251111 (Git shallow)
Build date: 2025/11/11 16:45:10
Build type: Release
Branch: main
Hash: 4fbb389ce1295e04a5ad7f3a63f68b63b4f4036c
Python 3.11.14, Qt 6.8.3, Coin 4.0.3, Vtk 9.3.1, boost 1_86, Eigen3 3.4.0, PySide 6.8.3
shiboken 6.8.3, xerces-c 3.3.0, IfcOpenShell 0.8.2, OCC 7.8.1
Locale: Polish/Poland (pl_PL)
Navigation Style/Orbit Style/Rotation Mode: CAD/Rounded Arcball/Window center
Stylesheet/Theme/QtStyle: unset/FreeCAD Light/fusion
Logical DPI/Physical DPI/Pixel Ratio: 96/90.0633/1
```

**Comment by furgo16** [comment]
The tab order is not defined in the .ui file, in which case the default is to use the order in which the widgets are defined in the file. Since they're not defined sequentially in the way they are rendered, then the tab order is all over the place. Here's what it looks like in Qt Designer's Tab Order Editor.

<img width="380" height="410" alt="Image" src="https://github.com/user-attachments/assets/b2dc469e-2d56-4499-9ee5-fc044d0e3299" />

**Comment by leoheck** [comment]
Thanks @furgo16. Other panels and forms are also difficult to use with the keyboard. We should keep reporting these issues.

**Comment by furgo16** [comment]
Indeed, feel free to. Happy to tackle them if I can.
